### ISSUE TITLE
Menus don't highlight when navigated using arrow keys

### ISSUE THREAD

**Issue by DanielPiersonLN** [notepad-plus-plus/notepad-plus-plus#13688]
### Description of the Issue
Using keyboard shortcuts to open menus, when navigating the menus with arrow keys, the menu items don't highlight.

### Steps to Reproduce the Issue
1. Open an empty document
2. Type Alt-E to acitvate the Edit Menu
3. Type down arrow

### Expected Behavior
Menu items highlight as you arrow down through the menu.

### Actual Behavior
Menu items don't highlight.

### Debug Information
Notepad++ v8.5.3   (64-bit)
Build time : May 15 2023 - 06:09:36
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "C:\Program Files\Notepad++\change.log" 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 Enterprise (64-bit)
OS Version : 22H2
OS Build : 22621.1702
Current ANSI codepage : 1252
Plugins : 
    ComparePlugin (2.0.2)
    HexEditor (0.9.12)
    JSMinNPP (1.2202)
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4)
    NppFavorites (1.0.0.1)
    NPPJSONViewer (1.40)
    NppSnippets (1.7)
    PoorMansTSqlFormatterNppPlugin (1.6.13.31508)
    XMLTools (3.1.1.13)

- Menu items are getting activated as when you press enter, the active item is invoked
- Also, right arrow to navigate sub-menus works as expected, you just can't see the highlight so it is hard to know when you're on the right parent menu.
- See attached screen capture.
- This was working as expected before.  Only seem to have broken in the last release... not sure how long ago.  Has been a few weeks anyway.

https://github.com/notepad-plus-plus/notepad-plus-plus/assets/32872467/305da643-8b86-46b9-a635-bbbb77a47331

**Comment by DanielPiersonLN** [comment]
Additional Info... I tried starting with -noPlugin to see if one of the plugins was involved but got the same behavior.

**Comment by alankilborn** [comment]
In general, I don't see this same behavior in 8.5.3.

However, I noted some points:

* In dark mode, there is no visual highlight on disabled menu items.  Thus, with no active selection when Alt+e is pressed, it takes 4 down-arrow presses before anything visually changes (when _Paste (enabled) gets highlighted).  @ozone10, as this references "dark mode", you may have interest.

* Why do disabled items even ever get a highlight when you up and down arrow through a dropped menu -- you can't choose them, so what's the point??

**Comment by alankilborn** [comment]
ozone10 said:

>Not a Notepad++ issue, this is how MS implemented it.

I presume you mean disabled items getting a highlight (my second bullet point).
So, OK on that.

Regarding my first bullet point:
But I see, in dark mode, no visual highlight on disabled menu items.  If I see a visual highlight on them in light mode, shouldn't I see one in dark mode?

**Comment by ozone10** [comment]
@alankilborn 
nope, I mean both are MS implementation. Right-click on desktop (on Windows 11 open old context menu) with paste disabled and press arrow keys.
